docs/ur/README.md

```diff
@@ -0,0 +1,351 @@
+
+```
+
+Example:- Sync
+Wrong: ہم وقت ساز
+Correct: سنک
+
+Example: Security
+Wrong: سلامتی
+Correct: سیکورٹی
+
+```
+
+* **Localization**: Localization means that the translated text needs to be adapted to the to the local language, customs and localization standards. For example, in many cases you would need to use Urdu names rather than English
+* **Translation**: Translation is taking the meaning of a word from the source text and providing an equivalent text in the target language.
+* **Transliteration (Transliterate)**: Transliteration is converting the text from one script to another. It does not render meaning. This means changing only the source letters or characters into corresponding those of the target language.
+
+```
+
+Example:- New York
+Wrong: نیا یارک
+Correct: نیو یارک
+
+Example: Damascus
+Wrong: دماسکس
+Correct: دمشق
+
+```
+
+Do not translate every word, but use the style that is natural for Urdu.
+
+* **Consistency**: Please ensure that all terminology is used consistently both within one component and across different components (software, help, documentation). In most cases terminology needs to be consistent also across different products. Moreover please use consistent style and register and translate similar phrases consistently.
+
+Refer [Glossary](glossary.md)
+
+## Formality and tone
+
+### Follow these basic rules:
+
+* Original American English text tends to be rather casual. For the Urdu language, the general style should be clear, friendly and concise. Being friendly does not mean using overly colloquial language – it is crucial to consistently maintain a professional tone, but use contemporary, up-to-date style and common words. Use language that resembles conversation observed in everyday settings as opposed to the formal, technical language that is often used for technical and commercial content.
+* Try to avoid long, nested sentence constructions. If necessary, break up the original sentence and regroup it syntactically.
+
+```
+
+Example: Fast for Good.
+Wrong: اچھے کے لئے تیز۔
+Correct: تیز، بہتری کے لئے۔
+
+```
+
+* Use wording that is succinct, unambiguous, and free of jargon.
+* Produce a translation that sounds as it if was originally written in your language, i.e. avoid following the original source sentence structure too closely.
+* Always bear in mind who your target audience is (i.e. an experienced computer user, a beginner, or a combination of both groups).
+* Use a consistent style throughout all product components and across a product range, to ensure that all Mozilla products can be linguistically identified as part of a group of products.
+* Use Formal expressions.
+
+```
+
+Example: Your account is safe
+Wrong: تمہارا اکاؤنٹ محفوظ ہے
+Correct: آپ کا اکاؤنٹ محفوظ ہے
+
+```
+
+## Reference Terminology
+
+The following terminology sources should be used as reference in the translation:
+* Urdu [Glossary](glossary.md)
```

Comment:
Note that this year, Pontoon will have its own glossary functionality, so at some time, this will need to be migrated and the page removed.

Reply:
Ok. Got it. Then we can move glossary.